README.md

```diff
@@ -19,74 +17,57 @@ Allows you to control your 3D Printer, like you can do with any [TFT/LCD panel](
 
 ### How this is different from TouchUI?
 
-[TouchUI](http://plugins.octoprint.org/plugins/touchui/), is an amazing plugin
-for Octoprint, was created as a responsive design for access to OctoPrint,
-from low resolution devices, such as smartphones, tablets, etc.
+[TouchUI](http://plugins.octoprint.org/plugins/touchui/) is an amazing plugin for Octoprint.  It was created as a responsive design for access to OctoPrint from low resolution devices such as smartphones, tablets, etc.
 
-Executing TouchUI under a RPi w/TFT modules, presents two big problems,
-first isn't optimized to be used with resistive touch screens with low resolutions
-like 480x320 and second requires a browser to be access, consuming a lot of
-resources.
+Executing TouchUI under a RPi w/TFT modules presents two big problems:
+1. It is not optimized to be used with resistive touch screens with low resolutions like 480x320
+2. It requires a browser to be used for access, consuming extra resources.
 
-This is the main reason because I develop this X application to be executed
-in my 3d printer.
+For these reasons I developed this X application to be used with my 3d printer.
 
-Installation
-------------
+Basic Installation
+------------------
 
-### Dependencies
+(If you are having difficulty getting this working with your TFT, please see the wiki page: [TFT Installation Tips](https://github.com/CJeffyB/OctoPrint-TFT/wiki/TFT-Installation-Tips))
```

Comment:
Can you move it to the bottom of the installation section?

Reply:
Done.